docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md

```diff
@@ -1,8 +1,17 @@
 Using the ArangoDB Starter
 ==========================
 
-This section describes how to start an Active Failover setup the tool [_Starter_](../../Programs/Starter/README.md)
-(the _arangodb_ binary program).
+This section describes how to start an _Active Failover_ setup using the tool
+[_Starter_](../../Programs/Starter/README.md) (the _arangodb_ executable).
+
+{% hint 'warning' %}
+It is **strongly suggested** to **not** run the _ArangoDB Starter_ process
+(_arangodb_) from a system _user_ with full privileges, e.g. _root_ under Linux.
+
+We suggest to create a dedicated _user_ with limited privileges for this purpose.
+Please refer to the [Linux user used to run _arangodb_](../ArangoDBStarter/README.md#linux-user-used-to-run-arangodb)
+section of the Starter Deployment page for further details.
```

Comment:
Can we add some info for macOS and Windows as well?

Reply:
Simran - can you?

Reply:
On Windows in a Command Prompt or PowerShell you can do `runas /user:arangodb "arangodb args"` but I couldn't figure out how to escape certain characters. Is our general recommendation to use a restricted system user account on all platforms? I know that MySQL creates a separate (hidden?) account, which has different file & folder permissions for instance. I don't know what type of account it actually is however.

Reply:
please can you open a ticket for this? we incorporate it in a later PR - thanks